lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -287,6 +302,24 @@ class XMLTemplateAnalyzer {
 		}
 	}
 
+	_analyzeCoreRequire(node) {
+		const coreRequire = getAttributeNS(node, XMLVIEW_CORE_REQUIRE_ATTRIBUTE_NS);
+		let requireContext;
+		if ( coreRequire ) {
+			try {
+				requireContext = JSTokenizer.parseJS(coreRequire);
+			} catch (e) {
+				log.error("Require attribute can't be parsed on Node: ", node.$ns.local);
+				throw e;
+			}
+			if (requireContext) {
```

Comment:
in the runtime code, we have added some validation for the object literal (keys must be simple identifiers, values must be string literals). At least for the requireJsName, a check for string literal might make sense...

Reply:
Good point 👍
Tests for those cases should also be added.

Reply:
Done